nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -137,41 +138,60 @@ class ConformerConvolution(nn.Module):
     Args:
         d_model (int): hidden dimension
         kernel_size (int): kernel size for depthwise convolution
+        pointwise_activation (str): name of the activation function to be used for the pointwise conv.
+            Note that Conformer uses a special key `glu_` which is treated as the original default from
+            the paper.
     """
 
-    def __init__(self, d_model, kernel_size, norm_type='batch_norm'):
+    def __init__(self, d_model, kernel_size, norm_type='batch_norm', pointwise_activation='glu_'):
```

Comment:
Why it is not just glu?

Reply:
So the string "glu" can be activation without the doubling of filters in the dw_conv_input_dim, but for backward compact we want to support "glu" but without the double channel (original conformed) - so _ is to mark it as the default used by conformed (originally it cant be modified by config so it should be safe).